src/components/atoms/SfRadio/README.md

```diff
@@ -0,0 +1,25 @@
+# SfRadio
+
+Component for simple group of radio buttons, pass an array get selected value via v-model
+
+## Events
+
+- `input` - event emmited when option is selected
+
+<!-- Write down available CSS Modifiers -->
+
+## SCSS variables
```

Comment:
just get rid of this one to not duplicate and I can marge ;)

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/91/commits/a56bc8e3c3d4610730d25da8c9f08c398541d31a